prow/cmd/deck/static/tidescript.js

```diff
@@ -227,6 +227,21 @@ function createPRCell(pool, prs) {
     return c;
 }
 
+function createBatchCell(pool) {
+    var td = document.createElement('td');
+    if (pool.BatchPending) {
```

Comment:
`[]` is truthy so this should probably be `if (pool.BatchPending && pool.BatchPending != [])`.